src/main/groovy/com/palantir/gradle/docker/DockerRunExtension.groovy

```diff
@@ -30,7 +30,7 @@ class DockerRunExtension {
     private List<String> command = ImmutableList.of()
     private Set<String> ports = ImmutableSet.of()
     private Map<String,String> env = ImmutableMap.of()
-    private Map<String,String> volumes = ImmutableMap.of()
+    private Map<Object,String> volumes = ImmutableMap.of()
```

Comment:
Actually would prefer to just leave this as a string also. Not sure there is any value in supporting file objects.

Reply:
The `project.file()` behavior is very much the standard in Gradle.